configs/mmrotate/rotated-detection_sdk_dynamic.py

```diff
@@ -0,0 +1,8 @@
+_base_ = ['./rotated-detection_static.py', '../_base_/backends/sdk.py']
+
+codebase_config = dict(model_type='sdk')
+
+backend_config = dict(pipeline=[
+    dict(type='LoadImageFromFile'),
+    dict(type='Collect', keys=['img'], meta_keys=['filename', 'ori_shape'])
```

Comment:
But In the postprocess module `RResizeBBox`, it requires `ori_shape` and `scale_factor` from `img_metas`

Reply:
ori_shape and scale_factor information should be added by resize module.